src/store/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod integer;
+pub mod integermulti;
+pub mod multi;
+pub mod single;
```

Comment:
Nit: per [No more mod.rs](https://rust-lang-nursery.github.io/edition-guide/rust-2018/module-system/path-clarity.html#no-more-modrs), this file should be located at _src/store.rs_.